server/src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -966,19 +977,12 @@ public boolean isForceExecution() {
                         }
 
                         @Override
-                        public void onFailure(Exception e) {
-                            handleSendToLocalException(channel, e, action);
-                        }
+                        public void onFailure(Exception e) {}
 
                         @Override
                         public String toString() {
                             return "processing of [" + requestId + "][" + action + "]: " + request;
                         }
-
-                        @Override
-                        public void onAfter() {
-                            request.decRef();
```

Comment:
Where did this go, regardless of the fact that we need to do this differently, it seems we lost the `decRef` here?